apis/config/scheme/scheme_test.go

```diff
@@ -720,7 +720,6 @@ leaderElection:
   retryPeriod: 0s
 metricsBindAddress: ""
 parallelism: 0
-percentageOfNodesToScore: 0
```

Comment:
this field got converted to `*int64` which default value is `nil` and it uses `omitempty` tag